### ISSUE TITLE
Create the functionality `save to svg`

### ISSUE THREAD

**Issue by ThomAub** [procyon-rs/procyon#2]
In order to save the chart render one can use web browser opened and click on `save png`or `save svg`. 
in the Altair documentation they use selenium to bypass the manual part and save directly.
It would be nice to have this feature also. see here for [altair documentation]( https://altair-viz.github.io/user_guide/saving_charts.html?highlight=saving) 

A selenium crate seems to exist and thus the same technic could be used : [selenium-rs](https://crates.io/crates/selenium-rs)

**Comment by ChristianBeilschmidt** [comment]
That sounds great :+1:. I guess it makes sense to put it into a common crate rather than to duplicate functionality. I'm looking forward to using it.

So you basically use a headless Firefox for rendering the charts and returning the SVG? Is there a way to keep the browser instance open? I'm not quite sure what the performance penalties are for starting the process over again.

**Comment by ThomAub** [comment]
> So you basically use a headless Firefox for rendering the charts and returning the SVG?

Yes that's exactly it. 
> Is there a way to keep the browser instance open?

Yes it's possible this mean that you want to save multiple charts ?

**Comment by ChristianBeilschmidt** [comment]
> > Is there a way to keep the browser instance open?
> 
> Yes it's possible this mean that you want to save multiple charts?

Yeah, I plan to investigate it for a server process. For histograms, it is good to transfer it to a client browser and have an interactive plot but for scatter plots with lots of points you not necessarily want to transfer all data to the client.

**Comment by ChristianBeilschmidt** [comment]
Did you manage to make some progress on this issue or maybe push your current state to a public branch?

**Comment by ThomAub** [comment]
Hello, 
Yes i managed to have something even if it's a bit ugly .. I will ask for review of another contributor and push it to a public branch in the following days.
